configure.ac

```diff
@@ -536,6 +536,25 @@ fi
 
 
 # MATH LIBRARY SELECTION
+
+# no math library
+AC_ARG_ENABLE([nomath],
```

Comment:
Did you also consider `./configure --disable-sp-math-all --disable-fastmath --disable-heapmath  --enable-cryptonly --disable-rsa --disable-ecc --disable-dh --disable-asn && make`?
If you would like to keep `nomath` consider adding a test case for it.

Reply:
I agree and do not like adding in another --enable- option. Having instead the --disable-(sp-math-all | fastmath | heapmath). In the case of user_settings.h builds would it remain where it automatically has (sp-math-all) though?

Reply:
Reading this comment and getting back to the PR I think we should keep the WOLFSSL_NO_MATH macro and have configure.ac set it if all math libraries are disabled. That way empty user_settings.h will use sp-math-all (easy default setup) and there is still a way for configure and user_settings.h builds to turn off all math libraries without a new --enable- flag.